tests/foreman/api/test_organization.py

```diff
@@ -217,6 +222,8 @@ def test_positive_update_description(self, module_org, desc):
         :expectedresults: The organization's description is updated.
 
         :CaseImportance: Critical
+
+        :parametrized: yes
         """
         setattr(module_org, 'description', desc)
         module_org = module_org.update(['description'])
```

Comment:
There's a parametrization on line 326 in the file that should include a docstring that's missing.

Reply:
great catch, thanks! Added.